.buildkite/pipeline.yaml

```diff
@@ -74,6 +74,7 @@ steps:
     key: "playwright-step"
     command: ".buildkite/scripts/run-playwright.sh"
     timeout_in_minutes: 15
+    branches: "!master !canary"
```

Comment:
Nice, didn't know you could do this without a logical operator but seems like you can add as many filters as you like #theMoreYouKnow 👍 
![Screen Shot 2022-09-27 at 4 44 46 pm](https://user-images.githubusercontent.com/36558508/192452696-8ae6e972-8bc6-49a4-93ff-2eb00079c222.png)